dozer-api/src/generator/protoc/generator/implementation.rs

```diff
@@ -360,7 +360,9 @@ impl Names {
 fn convert_dozer_type_to_proto_type(field_type: FieldType) -> Result<String, GenerationError> {
     match field_type {
         FieldType::UInt => Ok("uint64".to_owned()),
+        FieldType::U128 => Ok("uint128".to_owned()),
```

Comment:
This function must be consistent with `field_to_prost_value`.

Reply:
yup